### DISCUSSION TITLE
Verhinderung von Zoom und Pan über Extent

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#23] [Q&A]
Wie kann man Zoom und Pan über den Extent hinaus verhindern?

**Comment by gis-eni** [comment]
Kommt ab Build 5.23.601. Es muss allerdings auch dann über die ``custom.js`` aktiviert werden:

``` javascript
webgis.usability.zoom.setMaxBounds = true;
```
